subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -1109,8 +1109,12 @@ static inline void ticker_job_node_manage(struct ticker_instance *instance,
 		ticker->req = ticker->ack;
 
 		if (instance->ticker_id_slot_previous == user_op->id) {
+			u32_t ticks_now = cntr_cnt_get();
+
 			instance->ticker_id_slot_previous = TICKER_NULL;
-			instance->ticks_slot_previous = 0U;
+			instance->ticks_slot_previous = ticks_elapsed +
```

Comment:
should this be ```MIN(ticker->ticks_slot, (ticks_elapsed + ticker_ticks_diff_get(ticks_now, instance->ticks_current)))``` ?

Reply:
@mtpr-ot do review when possible, this is a critical bug fix.

Reply:
> should this be `MIN(ticker->ticks_slot, (ticks_elapsed + ticker_ticks_diff_get(ticks_now, instance->ticks_current)))` ?

I think doing the MIN is right.

Reply:
> @mtpr-ot do review when possible, this is a critical bug fix.

It looks good. I am testing here, and when the MIN is added and tests passed, I will approve.

Reply:
> > @mtpr-ot do review when possible, this is a critical bug fix.
> 
> It looks good. I am testing here, and when the MIN is added and tests passed, I will approve.

Testing passes here.